packages/ERTP/src/mathHelpers/natListMathHelpers.js

```diff
@@ -0,0 +1,43 @@
+import harden from '@agoric/harden';
+import { passStyleOf } from '@agoric/marshal';
+import { assert } from '@agoric/assert';
+import Nat from '@agoric/nat';
+
+// Operations for arrays with unique Nat elements.
+const identity = harden([]);
+
+const natListMathHelpers = harden({
+  doAssertKind: list => {
+    assert(passStyleOf(list) === 'copyArray', 'list must be an array');
+    list.forEach(Nat);
+  },
+  doGetIdentity: _ => identity,
+  doIsIdentity: list => passStyleOf(list) === 'copyArray' && list.length === 0,
+  doIsGTE: (left, right) => {
+    const leftSet = new Set(left);
+    const has = elem => leftSet.has(elem);
+    return right.every(has);
```

Comment:
This is a case where I at least would have understood it better if the function were written inline:

```js
return right.every(elem => leftSet.has(elem));
```

YMMV. If you find the status quo clearer, don't change it. Maybe the problem is solved by renaming `has` into `leftHas`.

Reply:
I like `leftHas`